SchemaInventory.json

```diff
@@ -5189,5 +5189,35 @@
       "dynamic": "No",
       "approved": "Yes"
     }
+  ],
+  "CifSurvey": [
+    {
+      "name": "CifSurvey",
+      "path": "Domains\\4-Application\\Connectors\\CivilInfrastructureFramework\\Released\\CifSurvey.01.00.00.ecschema.xml",
+      "released": true,
+      "version": "01.00.00",
+      "comment": "New schema for SurveyTools",
+      "verifier": "Colin.Kerr",
+      "sha1": "54af3904da9ce082458c26af7c35ef4c5c586644",
+      "verified": "Yes",
+      "author": "Monmohan.Bordoloi",
+      "date": "8/30/2024",
+      "dynamic": "No",
+      "approved": "Yes"
+    },
+    {
+      "name": "CifSurvey",
+      "path": "Domains\\4-Application\\Connectors\\CivilInfrastructureFramework\\Released\\CifSurvey.01.00.00.ecschema.xml",
```

Comment:
You have to provide the path of work in progress schema here including the correct file name at the moment it points to the released version.

Reply:
Created the WIP schema path